### PR TITLE
Update package.json dependencies and node version to >= 12

### PR DESCRIPTION
**Pull request by skphi13**
- Update minimum node version to node 12.
- Update the dependencies to their node 12 version

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -64,11 +64,11 @@
     "mocha": "^7.0.0"
   },
   "dependencies": {
-    "@hapi/crumb": "^7.2.3",
-    "@hapi/hoek": "^8.0.2",
-    "@hapi/joi": "^16.1.7",
+    "@hapi/crumb": "^8.0.0",
+    "@hapi/hoek": "^9.0.4",
     "@vrbo/steerage": "^8.4.0",
```
Comment (tuckbick) [inline package.json]: Probably ought to update [steerage](https://github.com/expediagroup/steerage) first.

File: package.json [hunk 2]
```diff
@@ -64,11 +64,11 @@
     "mocha": "^7.0.0"
   },
   "dependencies": {
```
Comment (tuckbick) [inline package.json]: Just as a matter of consistency, could you move `dependencies` above `devDependencies`?

**Review by tuckbick — CHANGES_REQUESTED** [review]

File: package.json [hunk 3]
```diff
@@ -52,10 +52,10 @@
   ],
   "license": "Apache-2.0",
   "peerDependencies": {
-    "@hapi/hapi": ">=18.3.1"
+    "@hapi/hapi": ">=19.2.0"
```
Comment (ianwhitedeveloper) [inline package.json]: We should loosen this to `">=19.0.0", unless there is a specific reason to require `19.2.0`

**Review by ianwhitedeveloper — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (tuckbick) [reply package.json]: Should probably go back to catalyst-render too to ensure it's consistent there too.

**Review by tuckbick — CHANGES_REQUESTED** [review]
Per the [Hapi 19 release notes](https://github.com/hapijs/hapi/issues/4017), I think we also need to set Joi as the validator: https://hapi.dev/api/?v=19.2.0#-servervalidatorvalidator

**Review by tuckbick — APPROVED** [review]

**Pull request merged**